pkg/controller/controller.go

```diff
@@ -226,25 +229,74 @@ func (c *Controller) initResource() (string, error) {
 			return "", fmt.Errorf("fail to create CRD: %v", err)
 		}
 	}
+
+	err = c.createPSConfigMap()
+	if err != nil {
+		log.Errorf("createPSConfigMap() returned error: %v", err)
+	}
+
 	return watchVersion, nil
 }
 
+//Create a ConfigMap containing the source for a simple grpc server (pkg/controller/grpc_tensorflow_server.py)
+//that will be used as default PS
+func (c *Controller) createPSConfigMap() error {
+	//If a ConfigMap with the same name already exists, it was created by an earlier operator
+	//we delete and recreate it in case the grpc_tensorflow_server.py was updated in the meantime
+	cm, err := c.KubeCli.CoreV1().ConfigMaps(c.Namespace).Get(spec.PSConfigMapName(), metav1.GetOptions{})
```

Comment:
Good catch.
1 ConfigMap per job would also mean specifying the volumes configuration for the default PS in the create phase (probably where I specify the Command) because the name of the ConfigMap won't be known in advance anymore and will rely on the RuntimeId.
Are you ok with that?

Reply:
SGTM.